test_suites/h264/JVT-AVC_V1.json

```diff
@@ -1017,15 +1017,15 @@
             "source_checksum": "23c47c4535e838cb2ac4e98eb05c9238",
             "input_file": "sp1_bt_a.h264",
             "output_format": "yuv420p",
-            "result": "ca0b3580f448cc1e07f81a4756273b59"
+            "result": "c086a2dd4a002734ee4d385a797a3536"
         },
         {
             "name": "sp2_bt_b",
             "source": "https://www.itu.int/wftp3/av-arch/jvt-site/draft_conformance/AVCv1/sp2_bt_b.zip",
             "source_checksum": "c1601a80dabffd65a1c3416635b6e862",
             "input_file": "H26L/BitstreamExchange/sp2_bt_b.h264",
             "output_format": "yuv420p",
-            "result": "77fef826070d3d17174529b16765bb24"
+            "result": "6de46a828ea40399b0741efd0bdfea11"
```

Comment:
The reference decoder does not work correctly for these two, apparently. We knew about this. We took the results from the .zip themselves.